File: openapi-generator/dart-jaguar/lib/model/inline_object19.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'inline_object19.jser.dart';

class InlineObject19 {
   /* Group ID to query. */
  @Alias('groupId')
  final int groupId;
   /* List of sensor IDs to query. */
  @Alias('sensors')
  final List<int> sensors;
  

  InlineObject19(
      

{
    
     this.groupId = null,  
     this.sensors = const [] 
    }
  );

  @override
  String toString() {
    return 'InlineObject19[groupId=$groupId, sensors=$sensors, ]';
  }
}

@GenSerializer(nullableFields: true)
class InlineObject19Serializer extends Serializer<InlineObject19> with _$InlineObject19Serializer {

}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/InlineObject19.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param groupId Group ID to query.
 * @param sensors List of sensor IDs to query.
 */
data class InlineObject19 (

        @get:NotNull 
        @JsonProperty("groupId") val groupId: Long,

        @get:NotNull 
        @JsonProperty("sensors") val sensors: List<Long>
) {

}

